routes/web.php

```diff
@@ -20,14 +20,26 @@
 
 Route::get('/home', 'HomeController@index')->name('home');
 
-Route::get('/search','SearchController@search')->name('search');
+Route::get('/informations-personnelles', 'UserController@edit')->name('user.edit');
+
```

Comment:
Route::get('/user/edit', 'UserController@edit')->name('user.edit'); 
would be better, same for the next route